core/tests/protractor_desktop/SkillEditorPage.js

```diff
@@ -0,0 +1,87 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview End-to-end tests for the skill editor page.
+ */
+
+var forms = require('../protractor_utils/forms.js');
+var general = require('../protractor_utils/general.js');
+var users = require('../protractor_utils/users.js');
+var waitFor = require('../protractor_utils/waitFor.js');
+var workflow = require('../protractor_utils/workflow.js');
+
+var AdminPage = require('../protractor_utils/AdminPage.js');
+var TopicsAndSkillsDashboardPage =
+  require('../protractor_utils/TopicsAndSkillsDashboardPage.js');
+var SkillEditorPage =
+  require('../protractor_utils/SkillEditorPage.js');
+
+describe('Skill Editor functionality', function() {
```

Comment:
What about skill publishing tests? Can you add that also?
As in, before publishing skills, it would be shown in 'Unpublished skills' section in dashboard, while after, it will be in 'Unused Skills' section. 
Also, test save draft functionality, i.e, save changes to skill, refresh or reload page and make sure changes persist.